ratatool-sampling/sampler.md

```diff
@@ -0,0 +1,24 @@
+Sampler
+=======
+
+Diffy contains record sampling classes for Avro, BigQuery, Parquet.
+
+# BigSampler
+
+BigSampler will run a [Scio](https://github.com/spotify/scio) pipeline sampling either Avro or BigQuery data.
```

Comment:
can we note that it supports `gs://` paths as well

Reply:
Added